ivadomed/scripts/training_curve.py

```diff
@@ -211,6 +182,55 @@ def run_plot_training_curves(input_folder, output_folder, multiple_training=Fals
         plt_dict[fname_out].savefig(fname_out)
 
 
+def tensorboard_retrieve_event(dpath):
+    """
+    Function that retrieves data from tensorboard summary event
+    Args:
+        dpath (str): log directory where the event are located
+
+    Returns:
+        df: a panda dataframe where the columns are the metric or loss and the row are the epochs.
+
+    """
+    list_metrics = ['dice_score', 'multiclass dice_score', 'hausdorff_score', 'precision_score', 'recall_score',
+                    'specificity_score', 'intersection_over_union', 'accuracy_score']
```

Comment:
Could you please add a TODO: something to remind us to find a way for not hardcoding this list? somehow.. thx

Reply:
Also add a note that elements of this list are in the same order as in the training file...

Reply:
done in e17adda